src/scripts/main.js

```diff
@@ -1,3 +1,73 @@
 'use strict';
 
-// write code here
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const table = document.querySelector('table');
```

Comment:
for table in table exist tbody element
select it 
```suggestion
const table = document.querySelector('tbody');
```